torchci/pages/api/drci/drci.ts

```diff
@@ -4,11 +4,97 @@ import fetchRecentWorkflows from "lib/fetchRecentWorkflows";
 import { RecentWorkflowsData } from "lib/types";
 import * as drciUtils from "lib/drciUtils";
```

Comment:
nit you should use something like
import {NUM_MINUTES, ...} from 'lib/drciutils';

The reason is if there's some script at the end of lib/drciUtils, it'll get executed when you import it like this.